packaging/docker/custom/Dockerfile

```diff
@@ -2,8 +2,32 @@ ARG GRAFANA_VERSION="latest"
 
 FROM grafana/grafana:${GRAFANA_VERSION}
 
+USER root
+
+ARG GF_INSTALL_IMAGE_RENDERER_PLUGIN="false"
+
+RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories && \
+    echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories && \
+    apk --no-cache update && \
+    apk --no-cache  upgrade && \
+    apk add --no-cache --virtual .build-deps udev ttf-opensans chromium && \
+    rm -rf /var/cache/apk/* /tmp/* && \
+    rm -rf /usr/share/grafana/tools/phantomjs; \
+fi
+
 USER grafana
 
+ENV GF_RENDERER_PLUGIN_CHROME_BIN="/usr/bin/chromium-browser"
+
+RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
+    grafana-cli \
+        --pluginsDir "$GF_PATHS_PLUGINS" \
+        --pluginUrl https://github.com/grafana/grafana-image-renderer/releases/latest/download/plugin-linux-x64-glibc-no-chromium.zip \
+        plugins install grafana-image-renderer; \
+fi

```

Comment:
Should you also remove the .build-deps virtual package?

Reply:
I thought the `--virtual .build-deps` only is used for installing the packages and doesn't leave any footprint. Could you suggest how I remove that?

Reply:
Removed the virtual since it didn't make sense here.